server/fishtest/rundb.py

```diff
@@ -522,18 +521,19 @@ def aggregate_unfinished_runs(self, username=None):
             if cores > 0:
                 eta = remaining_hours(run) / cores
                 pending_hours += eta
+                itp += run["args"].get("itp", 0)
```

Comment:
Change with zero practical effects but IMO it's more consistent if we use always the same default in case of missing key.

```python
itp += run["args"].get("itp", 100)
```

Reply:
Ah, I think my first version had this line elsewhere, but behind the `cores > 0` guard it does make sense to have a positive default.